src/components/CustomerQuota/CheckoutSuccess/RedeemedItem.tsx

```diff
@@ -12,26 +12,28 @@ export const RedeemedItem: FunctionComponent<{
   itemQuantities: ItemQuantities;
 }> = ({ itemQuantities }) => {
   const { getProduct } = useProductContext();
-  const { category, identifierInputs } = itemQuantities;
+  const { category, identifierInputs, quantities } = itemQuantities;
   const categoryName = getProduct(category)?.name ?? category;
 
   const identifierInputDisplay = identifierInputs
     ? getIdentifierInputDisplay(identifierInputs)
-    : "";
+    : null;
+
+  const quantitiesDisplay = Object.values(quantities)[0];
```

Comment:
Just wondering, will there be an edge case where quantities is null, or empty? Will this fail? :O

Reply:
based on the types, no